### DISCUSSION TITLE
Opening a schematic file (*.sch)

### DISCUSSION THREAD

**Discussion by mulkkyul** [robotology/community#124] [Q&A] [answered]
Hi all,

I have some problems with the MTB3_001 board (tactile sensor board) - (https://github.com/robotology/icub-support/issues/275), so I'm trying to get schematic of it. What I found on the svn repository is this one:
https://svn.robotology.eu/repos/iCubHardware-pub/trunk/electronics/public/MTB/Schematics/MTB3_REV1/

But I'm not able to open the schematic file (*.sch) . What kind of software should I use? (on Ubuntu or Windows)

Any tips would be appreciated.
Thanks.

**Comment by pattacini** [comment]
@maggia80 might help in this case, I guess.

**Comment by maggia80** [comment, marked as answer]
There is an error in the filename, but the pdf of the schematic is in the same folder and it is the MTB3-002.pdf

**Comment by mulkkyul** [comment]
Thanks! I checked it :)
